### DISCUSSION TITLE
Unable to verify

### DISCUSSION THREAD

**Discussion by GeSav** [community/community#55169] [GitHub Education]
### Hi everyone,

Hi everyone,

I am trying to verify for the Student Developer pack and I was horrified how BAD the verification process is, how it cannot be used for a large number of cases

1. I am a distance student and have no ID (as I do not go to the campus)
2. I printed out an official certificate from LADOK (the official source of all Swedish universities certificates etc) which clearly states that my type of studies is "distance learning" and Location is "Ortoberoende" (which means "independent of location" in swedish)
3.  I got an email that mentions possible reasons for rejection, none which is valid
---- I do have a matching name and a short bio
---- the school appears on the pdf I for so weird reason I have to print out and point to the camera instead of just uploading it as a file
---- the paper does mention expected participation, distance learning etc

What can I do?
I tried to highlight the part on the paper and re-apply with another photo and it says I cannot resubmit an application etc

This is really the worse interfaced. verification system I have ever experienced. I had no problem verifying through my university email to various other websites (Mathworks, Autodeskm Jetbrains etc) .

Is there a help line I could just talk to somebody like it is 1974 again ? or send an email like back in the 90s?

**Comment by GeSav** [comment]
+ One more thing ,I tried to take the picture with my mobile phone and it seems it cannot use the back camera, regardless which on I choose It always get the selfie one, So I cannot even get a decent photo with my phone and have to hold a paper in front of my laptop like a 70-year old grandpa

**Comment by CoderQiQin521** [comment]
good

**Comment by samus-aran** [comment]
Hi @GeSav 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

Thank you for bringing this to our attention regarding to the camera difficulties. We've passed this feedback to our internal team to investigate.

I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
